<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>update</class>
 <widget class="QDialog" name="update">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #FFFFFF;
    border-radius: 10px;
    border: 2px solid #1E90FF;
    font-family: Arial, sans-serif;
    color: #333;
}
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>411</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: white;
    color: #333;
	border: 2px solid black;
}
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: black;
    font-size: 16px;
    font-weight: bold;
	border:none
}
</string>
    </property>
    <property name="text">
     <string>New ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #F4F7FC;
    border: 1px solid #C5D1E0;
    border-radius: 5px;
    padding: 5px 10px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #1E90FF;
    background-color: #FFFFFF;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: black;
    font-size: 16px;
    font-weight: bold;
	border:none
}
</string>
    </property>
    <property name="text">
     <string>New Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #F4F7FC;
    border: 1px solid #C5D1E0;
    border-radius: 5px;
    padding: 5px 10px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #1E90FF;
    background-color: #FFFFFF;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: black;
    font-size: 16px;
    font-weight: bold;
	border:none
}
</string>
    </property>
    <property name="text">
     <string>New Quantity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #F4F7FC;
    border: 1px solid #C5D1E0;
    border-radius: 5px;
    padding: 5px 10px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #1E90FF;
    background-color: #FFFFFF;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #F4F7FC;
    border: 1px solid #C5D1E0;
    border-radius: 5px;
    padding: 5px 10px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #1E90FF;
    background-color: #FFFFFF;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: black;
    font-size: 16px;
    font-weight: bold;
	border:none
}
</string>
    </property>
    <property name="text">
     <string>Purchasing Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: black;
    font-size: 16px;
    font-weight: bold;
	border:none
}
</string>
    </property>
    <property name="text">
     <string>Selling Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #F4F7FC;
    border: 1px solid #C5D1E0;
    border-radius: 5px;
    padding: 5px 10px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #1E90FF;
    background-color: #FFFFFF;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updatebutton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #1E90FF;
    color: white;
    border-radius: 5px;
    border: none;
    padding: 8px 16px;
    font-size: 14px;
	font-weight:bold;
}

QPushButton:hover {
    background-color: #4682B4;
}

QPushButton:pressed {
    background-color: #4169E1;
}
</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
